src/main/java/com/mixpanel/android/mpmetrics/DecideChecker.java

```diff
@@ -263,6 +268,36 @@ private static int getDisplayWidth(final Display display) {
         }
     }
 
+    private static byte[] getUrls(RemoteService poster, Context context, String[] urls)
```

Comment:
what is the advantage you get out of moving `getUrls` from `RemoteService` to `Decidechecker`?